app/src/main/java/com/duckduckgo/app/brokensite/api/BrokenSiteSender.kt

```diff
@@ -83,6 +85,8 @@ class BrokenSiteSubmitter @Inject constructor(
                 featureToggle.isFeatureEnabled(PrivacyFeatureName.ContentBlockingFeatureName.value) &&
                 !contentBlocking.isAnException(brokenSite.siteUrl)
 
+            val vpnOn = runCatching { networkProtectionState.isRunning() }.getOrNull()
```

Comment:
why not `.getOrDefault(false)`?

Reply:
because `isRunning` failing doesn't mean VPN is disabled.

Reply:
Right. We want to differentiate that ( by setting null).